controls/radribbonview/features/simplified-ribbon.md

```diff
@@ -57,7 +57,11 @@ Before switching the layout, you have to define what will be shown in the simpli
 
 > The __SimplifiedResourses__ property of the __RadRibbonTab__ can be used in order to add any resources that should be applied when the LayoutMode is Simplified.
 
->tip In an MVVM scenario, the __SimplifiedItemTemplate__ property of the RadRibbonView needs to be set instead of the **ItemTemplate** property. Read more about that in the [Use MVVM in RadRibbonView]({%slug howto-use-mvvm-in-radribbonview%}) article. 
+## Databinding
```

Comment:
"Databinding" is not commonly used. In my opinion this is not correct, or at least it doesn't look good. Can we change this to "Data Binding"?